straxen/corrections_services.py

```diff
@@ -75,8 +77,8 @@ def get_corrections_config(self, run_id, config_model=None):
 
         if 'to_pe_model' in model_type:
```

Comment:
Is not `model_type` just a string? If yes we should change it in `==` as `"to_pe_model" in "to_pe_model_bla"` would be evaluate true too.

Reply:
no, since we have different detectors, see the following lines 

``` if 'to_pe_model' in model_type:                                                                                        
       # Get the detector name based on the requested model_type                                                          
       # This also will be used to the cachable name convention                                                           
       # pmt == TPC, n_veto == n_veto's PMT, etc                                                                          
       detector_names = {'to_pe_model': 'pmt',                                                                            
                                         'to_pe_model_nv': 'n_veto',                                                                      
                                         'to_pe_model_mv': 'mu_veto'}                                                                     
       target_detector = detector_names[model_type]```